index.bs

```diff
@@ -1082,6 +1082,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
         This is done in order to match the behavior of pervasively deployed ambient credentials (e.g., cookies, [[RFC6265]]).
         Please note that this is a greater relaxation of "same-origin" restrictions than what
         [=document.domain=]'s setter provides.
+
+        These restrictions on origin values apply to [=WebAuthn Clients=]. Other types of FIDO clients may use different origin URI formats, and have different rules for binding the origin URI to a [=Relying Party Identifier=], however the definition of these is outside the scope of this specification.
```

Comment:
>Other types of FIDO clients

I don't think we need to call out FIDO specifically here - there's (in theory) nothing FIDO-specific about non-WebAuthn integrations.

>may use different origin URI formats, and have different rules for binding the origin URI to a [=Relying Party Identifier=]

It doesn't necessarily need to be URIs, does it? I'm not actually familiar with how mobile apps are identified in app stores, for example, but I wouldn't assume they always have to be URIs.

With both of those things in mind, how about something like this?

```suggestion
        These restrictions on origin values apply to [=WebAuthn Clients=]. If other specifications mimic the [=Web Authentication API=] to make WebAuthn [=credentials=] usable on other platforms, they might have different rules for binding a caller to a [=Relying Party Identifier=]; they are outside the scope of this specification.
```